src/main/kotlin/org/partiql/scribe/targets/redshift/RedshiftAstToSql.kt

```diff
@@ -207,6 +207,11 @@ public open class RedshiftAstToSql(context: ScribeContext) : AstToSql(context) {
                 )
                 tail concat type("VARCHAR", 65535, gap = false)
             }
+
+            // Redshift does not honor the precision but store values with up to a maximum of six digits of precision for fractional seconds.
+            DataType.TIME, DataType.TIMESTAMP -> tail concat node.name()
+            DataType.TIME_WITH_TIME_ZONE -> tail concat "TIMETZ"
+            DataType.TIMESTAMP_WITH_TIME_ZONE -> tail concat "TIMESTAMPTZ"
```

Comment:
Small nit, my preference for consistency. Feel free to ignore. 

```suggestion
            DataType.TIME -> tail concat type("TIME")
            DataType.TIMESTAMP -> tail concat type("TIMESTAMP")
            DataType.TIME_WITH_TIME_ZONE -> tail concat type("TIMETZ")
            DataType.TIMESTAMP_WITH_TIME_ZONE -> tail concat type("TIMESTAMPTZ")
```